### ISSUE TITLE
Pesterchum says my computer denies it

### ISSUE THREAD

**Issue by slepyoneshot** [illuminatedwax/pesterchum#105]
I have had pesterchum for a while now and today it stopped working, it says when I try to open it "no connection could be made because the target machine actively refuses it"
please help me

-random person

**Comment by AKA-Grim** [comment]
Yeah, same, mine's been doing that for most of today (the 9th, my sleep schedule is boned) and the only thing that seems to work is restarting, but actually closing the program will not allow it to be reopened.

Advice would be rad.

**Comment by illuminatedwax** [comment]
This seems to be pretty common. It's some kind of network issue, but I
can't seem to recreate it at all. I'll check it out today.

**Comment by arcaneAgilmente** [comment]
There were a bunch of netsplits earlier yesterday which is what I thought
the issue was.

**Comment by SweaterDonningTree** [comment]
http://pastebin.com/eNN4Tsu8
the issue, apparently.
